src/nevergreen/config.clj

```diff
@@ -15,18 +17,21 @@
 (defn- invalid-key [aes-key]
   (throw (IllegalArgumentException. (str "An invalid AES_KEY was provided with length [" (count aes-key) "], a key length of [" aes-key-length "] is required"))))
 
-(defn port []
-  (Integer. (or (env :port) 5000)))
-
-(defn ip []
-  (or (env :ip) default-ip))
-
 (defn aes-key []
   (let [aes-key (env :aes-key)]
     (cond
       (nil? aes-key) @use-default-key
       (= aes-key-length (count aes-key)) aes-key
       :else (invalid-key aes-key))))
 
+(defn port []
+  (Integer. (or (env :port) default-port)))
+
+(defn ip []
+  (or (env :ip) default-ip))
+
 (defn log-level []
   (keyword (env :log-level)))
+
+(defn allow-iframe-from []
+  (or (env :allow-iframe-from) default-csp-frame-ancestors))
```

Comment:
We should probably add a new issue for validating (all) settings are valid, but I don't think it's required for this PR.